controller/employeeController.js

```diff
@@ -1,187 +0,0 @@
-const companytable = require("../models").company;
-
-
-//for creating new user 
-const createemployee = async (req, res) => {
```

Comment:
In javascript we usually use camelCase for naming variables, please use uniform casing through out the project :)

Reply:
most places i have used camelCase maybe forget to use at some cases :(